src/ray/raylet/node_manager.cc

```diff
@@ -1742,6 +1742,13 @@ void NodeManager::ProcessPushErrorRequestMessage(const uint8_t *message_data) {
 void NodeManager::HandleUpdateResourceUsage(rpc::UpdateResourceUsageRequest request,
                                             rpc::UpdateResourceUsageReply *reply,
                                             rpc::SendReplyCallback send_reply_callback) {
+  if (RayConfig::instance().use_ray_syncer()) {
+    RAY_LOG(INFO)
```

Comment:
Since we know the cause, I think we don't need to log an INFO, we can just add a comment explaining that when syncer is on, when this might be called.

Reply:
should we warn? this looks unexpected...

Reply:
Yes, I add this to warn the error but I don't want it to be pushed to the driver.  Will WARNING do this thing?

Reply:
If WARNING won't push the error to the user, I'm OK to change this.

Reply:
I think we should so that we can monitor it to make sure it's not happening in the future.

This could happen when the wrong setup is made and we should just let the user know this.